src/pkg/k8s/images.go

```diff
@@ -68,6 +69,11 @@ findImages:
 			return nil, fmt.Errorf("unable to get the node %s", pod.Spec.NodeName)
 		}
 
+		if nodeDetails.Status.Allocatable.Cpu().Cmp(requestedCPU) < 0 ||
+		   nodeDetails.Status.Allocatable.Memory().Cmp(requestedMemory) < 0 {
```

Comment:
```suggestion
		if nodeDetails.Status.Allocatable.Cpu().Cmp(minNodeCPU) < 0 ||
		   nodeDetails.Status.Allocatable.Memory().Cmp(minNodeMemory) < 0 {
```

These would need to be updated as well.

Reply:
Wow. No idea how I missed that. Thanks for the help!!!